project.inlang/README.md

```
@@ -20,28 +22,46 @@ Do not edit:
 
 Key files:
 
-- `settings.json` — locales, plugins, file patterns (source of truth)
+- `settings.json` — locales, plugins, file patterns
 - `cache/` — plugin caches (safe to delete)
 - `.gitignore` — generated
+- `README.md` — generated, explains this folder
+- `.meta.json` — generated SDK metadata
 
-```text
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)